introduction.md

```diff
@@ -1,2 +1,5 @@
 Hola blablabla mundo!!!
-HTML & CSS
+*HTML & CSS
+*Git
+*Javascript
+*Ruby
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?